lacework/resource_lacework_managed_policies.go

```diff
@@ -0,0 +1,135 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkManagedPolicies() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkManagedPoliciesCreate,
+		Update: resourceLaceworkManagedPoliciesUpdate,
+		Delete: resourceLaceworkManagedPoliciesDelete,
+		Read:   resourceLaceworkManagedPoliciesRead,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkManagedPolicies,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"policy": {
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "The id of the policy",
+							Required:    true,
+						},
+						"enabled": {
+							Type:        schema.TypeBool,
+							Required:    true,
```

Comment:
Is this field really required? 🤔

Reply:
If `enabled` is omitted, we should assume its value is true? Otherwise, would it be meaningless to manage `severity` for disabled policies? So would it be better to make this field required to explicitly specify its value instead of making assumptions? 

Another issue is that terraform SDK seems to always default optional boolean value to `false` if it's not specified in the `.tf` file.